man/homectl.xml

```diff
@@ -928,6 +929,27 @@
 
         <xi:include href="version-info.xml" xpointer="v245"/></listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><option>--session=</option><replaceable>SESSION</replaceable></term>
```

Comment:
name this --session-desktop= or --desktop= or so? or --preferred-desktop= or --preferred-session-desktop=.